### PR TITLE
Hide token in 'auth.gettoken' response

### PR DESCRIPTION
**Pull request by alexesprit**
It's better to hide token automatically by the extension instead of explaining people how to hide it manually (https://github.com/david-sabata/web-scrobbler/issues/981#issuecomment-238158879).

### CODE REVIEW

File: core/background/services/lastfm.js [hunk 1]
```diff
@@ -71,6 +69,10 @@ define([
 				// set token and reset session so we will grab a new one
 				data.sessionID = null;
 				data.token = xml.find('token').text();
+
+				var response = http_request.responseText;
+				console.log('getToken response: %s', response.replace(data.token, data.token.substr(5)));
```
Comment (inverse) [inline core/background/services/lastfm.js]: Why change it's position in the file?

Comment (alexesprit) [inline core/background/services/lastfm.js]: To prevent logging of response twice in error is occurred.

Comment (inverse) [inline core/background/services/lastfm.js]: In which case would it be better to use to use data.token as it's already parsed?

**Comment by inverse** [comment]
:+1:

**Pull request merged**